modules/installation-configuration-parameters.adoc

```diff
@@ -407,6 +407,20 @@ endif::ibm-power-vs[]
 Set the `networking.machineNetwork` to match the CIDR that the preferred NIC resides in.
 ====
 
+|networking:
+  ovnKubernetesConfig:
+    ipv4:
+      internalJoinSubnet:
+|When you configure `networkType` to `OVNKubernetes`, `OVNKubernetesConfig` provides the configuration for ovn-kubernetes as the default pod network.
```

Comment:
`OVNKubernetes` is the only supported `networkType` since `openshift-sdn` was removed, so I don't think we need this line. @patrickdillon do you know if any other network plugins are in the works or may be added some day, or are we safe to assume `OVNKubernetes` is going to be the only network plugin for now?

Reply:
Users are actually allowed to drop in other SDNs, such as calico.